mimesis/providers/internet.py

```diff
@@ -119,9 +119,8 @@ def emoji(self) -> str:
         """
         return self.random.choice(EMOJI)
 
-    @staticmethod
-    def image_placeholder(width: Size = 400,
-                          height: Size = 300) -> str:
+    # TODO: https://en.wikipedia.org/wiki/List_of_common_resolutions
```

Comment:
You can specify `todo`s in code: https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

Reply:
```rst
.. todo:: https://en.wikipedia.org/wiki/List_of_common_resolutions
```